tracer/test/test-applications/integrations/Samples.InstrumentedTests/Samples.InstrumentedTests.csproj

```diff
@@ -22,6 +22,7 @@
     <!-- as those force choosing the span overloads (which we currently don't support)   -->
     <!-- Once we add support for those overloads, we should remove this   -->
     <LangVersion>12.0</LangVersion>
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
```

Comment:
Is it really needed? Maybe an artefact but might not needed anymore?

Reply:
+1 I'm surprised to see this here since none of your updates to the test code seem to require it